plugins/http_plugin/http_plugin.cpp

```diff
@@ -217,7 +217,7 @@ class http_plugin_impl : public std::enable_shared_from_this<http_plugin_impl> {
          std::optional<eosio::chain::named_thread_pool>   thread_pool;
          std::atomic<size_t>                         bytes_in_flight{0};
          std::atomic<int32_t>                        requests_in_flight{0};
-         size_t                                      max_bytes_in_flight = 0;
+         uint64_t                                    max_bytes_in_flight = 0;
```

Comment:
This is later compared with a `size_t`, I think it is clearer as a `size_t`.

Reply:
Thanks. I will change back to size_t (was thinking make it explicitly 64 bits)